tests/tape/test_jacobian_tape.py

```diff
@@ -456,6 +456,7 @@ def test_choose_params_and_methods(self, diff_methods, argnum):
 
         assert len(res) == num_params
 
+    @pytest.mark.xfail
```

Comment:
Should this test be deleted if we've decided to remove the check altogether?

Reply:
Good idea!

Reply:
Done :)